tasks/util.py

```diff
@@ -1665,6 +1669,23 @@ def check_null_columns(self):
             raise ValueError('The following columns of the table "{table}" contain only NULL values: {columns}'.format(
                 table=self.output().table, columns=', '.join([x[0] for x in result])))
 
+    def check_universe_in_aggregations(self):
+        session = current_session()
+        result = session.execute("SELECT c.id, c.aggregate, cc.reltype "
+                                 "FROM observatory.obs_table t "
+                                 "INNER JOIN observatory.obs_column_table ct ON t.id = ct.table_id "
+                                 "INNER JOIN observatory.obs_column c ON ct.column_id = c.id "
+                                 "FULL JOIN observatory.obs_column_to_column cc ON c.id = cc.source_id "
+                                 "WHERE t.tablename = '{table}' "
+                                 "AND c.aggregate IN ('average', 'median') "
+                                 "AND (reltype IS NULL OR reltype <> 'universe')".format(
+                                     table=self.output()._tablename)).fetchall()
```

Comment:
I think this query also returns an error if an average/median column has a denominator. Does this case make sense?

I think it does (dividing an average of a class over the average of the total population, e.g: average income of engineers / average income).

Reply:
Currently, there isn't any case like this in the ETL but it makes sense, having a median/average with both a universe and a denominator shouldn's trigger the check.